lang/fr.js

```diff
@@ -47,7 +48,7 @@ export default {
 	"components.input-number.hintDecimalIncorrectComma": "Pour ajouter une décimale, utilisez la virgule « , »",
 	"components.input-number.hintDecimalIncorrectPeriod": "Pour ajouter une décimale, utilisez le point « . »",
 	"components.input-search.clear": "Effacer la recherche",
-	"components.input-search.defaultPlaceholder": "Recherche…",
+	"components.input-search.defaultPlaceholder": "Recherche en cours…",
```

Comment:
Is this right... @Mark-Weiss?  Not sure why this changed, but seems like it's trying to say a search is in progress which is not really true

Reply:
They both work. One's just being less formal, likely to save space. It amounts to "Searching..." to a more literal "Search in Progress..."

Reply:
Hmm, this is the placeholder text though so no search has happened yet.  I almost want like "Type your search here" or "What would you like to search?"... but obviously not those lol.  Is there a word that represents "search" instead of "searching", or is it it actually fine and just translating it to English is what makes it seem weird?

Reply:
Not 100% what's the best here, my speaking French to get my point across is much stronger than my "correct spelling". If it's just placeholder text just "Rechercher" would work. I'm using a bit of crowd sourced information from other apps. Here's what Facebook does:

![image](https://user-images.githubusercontent.com/23745288/127709319-641e6ffc-a646-4508-96e9-0c425ab37ce7.png)

Reply:
Yeah, I think I'm gonna put it back.  Thanks!